src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVKmerShort.java

```diff
@@ -106,40 +112,67 @@ public SVKmerShort canonical( final int kSize ) {
     public final int lastTrimer() { return (int)valLow & 0x3F; }
 
     @Override
-    public boolean equals( final Object obj ) {
-        return obj instanceof SVKmerShort && equals((SVKmerShort)obj);
+    public boolean equals(final Object obj) {
+        return obj instanceof SVKmerShort && equals((SVKmerShort) obj);
     }
 
-    public final boolean equals( final SVKmerShort that ) {
+    public final boolean equals(final SVKmerShort that) {
         return this.valLow == that.valLow;
     }
 
     @Override
     public final int hashCode() {
         // 32-bit FNV-1a algorithm
-        return fnvLong((int)2166136261L, valLow);
+        return SVUtils.fnvLong((int) 2166136261L, valLow);
     }
 
     /**
      * SVKmerShort comparison is consistent with equals.
      * It's also the same as the lexicographic ordering you'd get using toString on the Kmers.
      */
     @Override
-    public final int compareTo( final SVKmerShort that ) {
+    public final int compareTo(final SVKmerShort that) {
         return Long.compare(this.valLow, that.valLow);
     }
 
     /**
+     * Masks the current kmer by removing the specified bases.
+     * The mask array must be of even length, contain integers between 0 and kSize-1, and be in ascending order
+     * @param mask indices of masked bases
+     * @return kmer of length kSize - mask.length, with masked bases removed
+     */
+    public SVKmer mask(final byte[] mask, final int kSize) {
```

Comment:
My goof:  you can't have covariant args, just covariant return types.  You'll have to downcast an SVKmer.